packages/react-wallet/src/constants/chains.ts

```diff
@@ -1,6 +1,93 @@
 import { allChains as WagmiChains, Chain } from 'wagmi';
 
-export const FLAIR_CHAINS: Chain[] = WagmiChains;
+// TODO: Move these to wagmi repo itself
+// https://github.com/wagmi-dev/wagmi/blob/0.3.x/packages/core/src/constants/chains.ts#L234
+const newChains = [
+  {
+    id: 66,
+    name: 'OKC Mainnet',
+    nativeCurrency: { name: 'OKT', symbol: 'OKT', decimals: 18 },
+    rpcUrls: {
+      default: 'https://exchainrpc.okex.org',
+    },
+    blockExplorers: {
+      etherscan: {
+        name: 'Etherscan',
+        url: 'https://www.oklink.com/okc/',
+      },
+      default: {
+        name: 'default',
+        url: 'https://www.oklink.com/okc/',
+      },
+    },
+  },
+  {
+    id: 65,
+    name: 'OKC Testnet',
+    nativeCurrency: { name: 'OKT', symbol: 'OKT', decimals: 18 },
+    rpcUrls: {
+      default: 'https://exchaintestrpc.okex.org',
+    },
+    blockExplorers: {
+      etherscan: {
```

Comment:
@aramalipoor I couldn't fund the `etherscan blockExplorer` for `OKC` and `Avalanche`; and it was mandatory to put the value for it as pat of the chain object; so I used their own `blockExplorer` for both

Reply:
This is etherscan for Avalanche: https://snowtrace.io/

And for OKC you can put their in-house explorer `https://www.oklink.com/en/okc/` looks like their URLs are compatible with etherscan, better than nothing.

Reply:
Yep, added both :)